### ISSUE TITLE
Changing trainer name does not also change the name at the bottom right of the League Card.

### ISSUE THREAD

**Issue by BasedSquidKid** [kwsch/PKHeX#2447]
That goes for previously created cards, and making brand new cards at the rotomi machine too. Overall more of an oversight than a bug, but there isn't another issue open with this.

**To Reproduce**
1. Change Trainer Name in PKHeX
2. Inject Save into game
3. Open the menu and check League Card

**Expected behavior**
Name should be changed

**Screenshots**
![GameCapture_2019-11-18_01-39-32](https://user-images.githubusercontent.com/57887466/69030175-c9042500-09a4-11ea-866b-4751187989f4.jpg)
![GameCapture_2019-11-18_01-39-40](https://user-images.githubusercontent.com/57887466/69030176-c9042500-09a4-11ea-9a03-a46028350dac.jpg)
![GameCapture_2019-11-18_01-42-22](https://user-images.githubusercontent.com/57887466/69030177-c9042500-09a4-11ea-87b1-31e52b5a8cff.jpg)

**Comment by kwsch** [comment]
It's stored in multiple places in the save file. PKHeX only changes the main one.

**Comment by sora10pls** [comment]
Until this gets resolved, generating a new League Card using Rotomi (PC at the Pokémon Center) should fix the old name being there and replace it with the new one.

**Comment by BasedSquidKid** [comment]
> 
> 
> It's stored in multiple places in the save file. PKHeX only changes the main one.

So in that case, it would have been better to list it as a feature request. Either way, once all of the important legality/everything else is finished, would this be added?
